.rubocop.yml

```diff
@@ -2,25 +2,21 @@ require: rubocop-gitlab-security
 
 Layout/LineLength:
   Max: 200
-
 Layout/ParameterAlignment:
   Enabled: false
+Layout/SpaceBeforeBrackets: # (new in 1.7)
```

Comment:
Just curious, how did you determine which linters to add? Just want to know how we should update the linters in the future

Reply:
Rubocop is showing in the output which are the new linters missing in the configuration file.